pkg/opt/config.go

```diff
@@ -0,0 +1,124 @@
+package opt
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+
+	"cloud.google.com/go/bigquery"
+	vaultapi "github.com/hashicorp/vault/api"
+	"github.com/spf13/viper"
+	"google.golang.org/api/googleapi"
+)
+
+const (
+	DefaultListenPort = 50051
```

Comment:
Might want to pick a port <32768. Everything above is often used for ephemeral ports.